### ISSUE TITLE
Code for the Edit-Invariant Sequence Loss?

### ISSUE THREAD

**Issue by diegoantognini** [guangyliu/EISL#1]
Hi,

First, thank you for this great work to help replacing the cross-entropy loss for better generation.

I was wondering whether it was planned to release the pytorch/fairseq module of the loss, before the rest of your code.

Do you have a kind of an estimation when EISL will be available?

Thank you again for your work.

**Comment by guangyliu** [comment]
Thanks for your interest. We will clean and release the main code recently (maybe in one week.)

**Comment by diegoantognini** [comment]
Great, thank you for your quick answer! Good luck with your submission
